templates/decrypt-tls-assets

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash -e
+
+rkt run \
+	--volume=ssl,kind=host,source=/etc/kubernetes/ssl,readOnly=false \
+	--mount=volume=ssl,target=/etc/kubernetes/ssl \
+	--uuid-file-save=/var/run/coreos/decrypt-tls-assets.uuid \
+	--volume=dns,kind=host,source=/etc/resolv.conf,readOnly=true --mount volume=dns,target=/etc/resolv.conf \
+	--net=host \
+	--trust-keys-from-https \
+	quay.io/coreos/awscli:latest --exec=/bin/bash -- \
+		-ec \
+		'echo decrypting tls assets
+		shopt -s nullglob
+		for encKey in $(find /etc/kubernetes/ssl -name "*.pem.enc"); do
+		echo decrypting $encKey
+		f=$(mktemp $encKey.XXXXXXXX)
+		/usr/bin/aws \
+			--region {{.AWS.Region}} kms decrypt \
+			--ciphertext-blob fileb://$encKey \
+			--output text \
+			--query Plaintext \
+		| base64 -d > $f
+		mv -f $f ${encKey%.enc}
+	done;
+	echo done.'
```

Comment:
There is no standard for indentation of bash script, so maybe let's pick https://github.com/openstack-dev/bashate?